daemon/graphdriver/overlay2/overlay.go

```diff
@@ -345,10 +369,12 @@ func (d *Driver) Create(id, parent string, opts *graphdriver.CreateOpts) (retErr
 
 	if opts != nil && len(opts.StorageOpt) > 0 {
 		driver := &Driver{}
+
 		if err := d.parseStorageOpt(opts.StorageOpt, driver); err != nil {
 			return err
 		}
 
+		// if the quota Size is set then attempt to set it.
```

Comment:
This comment basically describes the comment directly in the `if` ("Set container disk quota limit"); I'd remove it because it's not really needed